libraries/AP_HAL_ChibiOS/hwdef/SkystarsF405DJI/hwdef.dat

```diff
@@ -0,0 +1,146 @@
+# Skystars F405 DJI
+# With F405 MCU, MPU6000 IMU and 7456 series OSD
+# Based on Mamba F405 MK2 from jeanphilippehell
+# Pin information from Betaflight
+
+# MCU class and specific type.
+MCU STM32F4xx STM32F405xx
+
+HAL_CHIBIOS_ARCH_F405 1
+
+# board ID for firmware load
+APJ_BOARD_ID 146
```

Comment:
This needs to be a unique number for this flight controller.  That means adding an entry to `Tools/AP_Bootloader/board_types.txt`, which you can do with a separate pull request - or another patch in here.

Reply:
Done!